src/main.py

```diff
@@ -76,6 +77,19 @@ async def get_forecasts_for_a_specific_gsp(
     return get_forecasts_for_a_specific_gsp_from_database(session=session, gsp_id=gsp_id)
 
 
+@app.get("/v0/forecasts/GB/pv/gsp_boundaries")
+async def get_gsp_boundaries() -> str:
+    """Get one gsp boundary for a specific GSP id
+
+    This is a wrapper around the dataset in
```

Comment:
maybe add a comment here (as it shows in the docs) that it's lat/lon EPSG:4326 as well